episodes/08-connected-components.md

```diff
@@ -345,29 +343,36 @@ print("min:", np.min(labeled_image))
 print("max:", np.max(labeled_image))
 ```
 
-Examining the output can give us a clue why the image appears black.
+Examining the output can give us a clue why the image appears empty.
 
 ```output
 dtype: int32
 min: 0
 max: 11
 ```
 
-The `dtype` of `labeled_image` is `int64`.
-This means that values in this image range from `-2 ** 63` to `2 ** 63 - 1`.
+The `dtype` of `labeled_image` is `int32`.
+This means that values in this image range from `-2 ** 31` to `2 ** 31 - 1`.
 Those are really big numbers.
 From this available space we only use the range from `0` to `11`.
 When showing this image in the viewer,
-it squeezes the complete range into 256 gray values.
-Therefore, the range of our numbers does not produce any visible change.
+it may squeeze the complete range into 256 gray values.
+Therefore, the range of our numbers does not produce any visible variation. One way to rectify this 
+is to explicitly specify the data range we want the colormap to cover:
 
-Fortunately, the scikit-image library has tools to cope with this situation.
+```python
+fig, ax = plt.subplots()
+plt.imshow(labeled_image, vmin=np.min(labeled_image), vmax=np.max(labeled_image))
```

Comment:
```suggestion
plt.imshow(labeled_image, vmin=np.min(labeled_image), vmax=np.max(labeled_image))
```
Since `fig` and `ax` are not being used?

Reply:
`fig, ax = plt.subplots()` is used here as I wanted to create a new figure rather than overwrite the previous one, its used without the course even when `fig` and `ax` are not used to I'm hesitant to change to an alterative.

Reply:
Oh, ok, thanks for explaining! So, I would suggest you keep this line but replace the next one with:
```py
ax.imshow(labeled_image, vmin=np.min(labeled_image), vmax=np.max(labeled_image))
```

Reply:
I agree this is better but its not consistent with how its done in the rest of the course so perhaps this should be a different PR where all similar examples are changed at once.

Reply:
Good catch, @mkcor. Please could you open a new issue where we can track this?